### ISSUE TITLE
Crypto Wallet extension suddenly fails with 404

### ISSUE THREAD

**Issue by DefiDebauchery** [brave/brave-browser#18082]
## Description 
At various times, the Crypto Wallet extension will become completely unavailable.
Strangely, only certain Browser Profiles will fail, while others do not.
Disabling and re-enabling the Wallet from the Extensions interface does not restore functionality.

## Steps to Reproduce

   1. Unknown, as it does not seem to happen in response to an action

## Actual result:
Popup:
![Screen Shot 2021-09-14 at 12 43 54 AM](https://user-images.githubusercontent.com/75273961/133197226-dfae8457-2c0b-40a9-9f2c-19ab0e4f550b.png)

Settings screen:
![Screen Shot 2021-09-14 at 12 53 54 AM](https://user-images.githubusercontent.com/75273961/133197228-c012f977-264d-4fb2-bb94-ba2662d5237f.png)

## Expected result:
Crypto Wallet/s never fails

## Reproduces how often: 
No way to specifically reproduce that I know of, but anecdotally it does appear to happen more often when a wallet page is open for an extended time (e.g. a website has invoked the login popup while away from the computer)

## Brave version (brave://version info)
Brave | 1.29.79 Chromium: 93.0.4577.63 (Official Build) (x86_64)
Revision | ff5c0da2ec0adeaed5550e6c7e98417dac77d98a-refs/branch-heads/4577@{#1135}
OS | macOS Version 11.5.2 (Build 20G95)

## Version/Channel Information:
Current

**Comment by srirambv** [comment]
Can you elaborate on how many profiles you have and does each profile have wallet created? Are you trying to use the wallets on all profiles at the same time? 

Does closing all profiles and restarting and launching `brave://wallet` update to the current stable release build?

**Comment by DefiDebauchery** [comment]
> Can you elaborate on how many profiles you have and does each profile have wallet created? 

I have about 12 profiles. Profiles that were not in use (and one profile that was in use but apparently either did not have Wallet loaded, or it actually did what it was supposed to and restart)  updated fine. Two that were actively using Wallets (including one that had generated an unlock popup while the machine was idle) did not. 

Most of them do have a Wallet; I keep a Profile per EVM chain (Ethereum, BSC, Polygon, etc) so I don't have to switch networks, which interrupts things I may be doing.

> Are you trying to use the wallets on all profiles at the same time?

I'm generally not using them at the same time; I will load a profile when needed and then close the window when I'm done.
 
> Does closing all profiles and restarting and launching `brave://wallet` update to the current stable release build?

Simply closing the windows of those profiles did not update the Wallet; restarting Brave as a whole of course did.

**Comment by srirambv** [comment]
Just circling back on this one sorry about the delay though. 

> Simply closing the windows of those profiles did not update the Wallet; restarting Brave as a whole of course did.

I did mean restart the entire Brave process by closing all the browser windows for all profiles. I assume the issue is fixed based on the above?

**Comment by DefiDebauchery** [comment]
Yes, I had to restart the entire Brave process in order to regain access to the wallet extension. But I wouldn't call the issue "fixed"; the extension shouldn't be failing when Brave tries to upgrade.

**Comment by DefiDebauchery** [comment]
Just a note that this happened again today, interrupting the ability to perform important transactions. I cannot restart Brave every time there's an update to the extension.
